ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -1030,14 +1030,27 @@ let vdi_copy_fun __context dbg vdi_map remote is_intra_pool remote_vdis so_far
         let id =
           Storage_migrate_helper.State.mirror_id_of (vconf.sr, vconf.location)
         in
-        debug "%s mirror_vm is %s copy_vm is %s" __FUNCTION__
+        let live_vm =
+          match Db.VDI.get_VBDs ~__context ~self:vconf.vdi with
+          | [] ->
+              failwith "VDI does not have a corresponding VBD"
```

Comment:
Should this included the specifics? Might want to define `failwith_fmt` to make this easier